lms/static/scripts/frontend_apps/errors.js

```diff
@@ -1,3 +1,16 @@
+/**
+ * @typedef {'reused_consumer_key'} AppLaunchServerErrorCode
```

Comment:
To check my understanding - this is an error that can happen in multiple types of frontend app launch, not just assignment launches?

Reply:
Correct. This is relevant when the app is put into `error-dialog` mode by the JS configuration object.